Activity.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import { Text, View, Button, Alert, Vibration } from "react-native";
+import { ActivitiesArray } from "./ActivitiesArray";
+import styled from 'styled-components/native'
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import { Lottie } from './Lottie';
+import { useFonts } from '@use-expo/font';
+import { AppLoading } from 'expo';
+
+export const Activity = ({ navigation }) => {
+  let [fontLoaded] = useFonts({
+    'Inter-Black': require('./assets/fonts/Inter-Black.otf'),
+  })
+  const [activity, setActivity] = useState()
```

Comment:
give this an initial state of either a random activity or a specific one, that way you won't get an empty screen at first when clicking on the virus.
```
const [activity, setActivity] = useState(ActivitiesArray[Math.floor(Math.random() * ActivitiesArray.length)])
or 
const [activity, setActivity] = useState(ActivitiesArray[0]) // to get the first one
```
The problem is that it's navigating to that screen before the activity is set with a new one.